_posts/2013-05-11-skip-nav-links.md

```diff
@@ -3,39 +3,52 @@ layout: post
 title: "How-to: Use skip navigation links"
 description: "Use skip nav links to ease keyboard user fatigue and frustration."
 author: cameron_cundiff
+last_updated: 2019-05-01
+updated_by: donavin_hannon
 published: true
 categories:
   - How-tos
 ---
-**Short answer:** Use skip nav links in conjunction with a coherent heading outline and [ARIA landmarks](https://a11yproject.com/posts/aria-landmark-roles/). You may need to also implement a JavaScript polyfill for Webkit based browsers.
+**Short answer:** Use skip navigation links in conjunction with a coherent heading outline and [ARIA landmarks](/posts/aria-landmark-roles/).
```

Comment:
Since all the explicit ARIA roles were removed from the code example, we should just mention "landmarks" per the following.

edit:
>heading outline and [ARIA landmarks](/posts/aria-landmark-roles/)

to:
>heading outline and [properly structured landmarks](/posts/aria-landmark-roles/).

TBH the title of the aria-lanmark-roles article should be updated as well.  but that's a separate issue I"ll address.